docs/learning/ui/appui/index.md

```diff
@@ -0,0 +1,34 @@
+# AppUi
+## Overview
+
+AppUi refers to an API and a set of components that enable iTwin.js apps to implement iTwin UI's App UI design. This design organizes the information  [iModel](../../iModels) data into various components on the screen. Graphical data is usually displayed in a [Viewport](../imodel&#8209;components/Viewport), while information such as element properties and data hierarchies are displayed in [Widgets](./appui&#8209;react/Widgets). An app using the App UI design will have a look similar to this:
```

Comment:
This reads a bit strange to me, "organizes the information iModel data into various components", should it possibly drop the "information" and just make it "organizes the iModel data into various components"?